app/user/rest.py

```diff
@@ -477,10 +480,13 @@ def get_organisations_and_services_for_user(user_id):
     return jsonify(data)
 
 
-def _create_reset_password_url(email):
+def _create_reset_password_url(email, next_redirect=None):
```

Comment:
Hey there, thanks for comments! Below my responses:

> next_redirect may not need a default argument of None. I think the only place we are using it, we always pass a value to it.

I agree that this argument would not need to be a default one, I will change that.

> param names are slightly different (next_redir vs next_redirect)

I can change param names in the `_create_2fa_url` to make it consistent with `next_redirect`, which I think is a better, more readable variable name.

> _create_2fa_url has email_auth_link_host to set the base_url, do we need to something similar?
notifications-api/app/utils.py

I don't think we need to add that to `_create_reset_password_url`, since it is not a new function, and it has not needed it so far and it works ☺️